common/src/test/java/com/microsoft/identity/common/internal/providers/microsoft/nativeauth/utils/MockApi.kt

```diff
@@ -0,0 +1,82 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+package com.microsoft.identity.common.internal.providers.microsoft.nativeauth.utils
+
+import com.google.gson.annotations.SerializedName
+import com.microsoft.identity.common.java.net.HttpConstants
+import com.microsoft.identity.common.java.net.UrlConnectionHttpClient
+import com.microsoft.identity.common.java.util.ObjectMapper
+import org.junit.Assert.assertTrue
+import java.net.URL
+import java.util.TreeMap
+
+class MockApi private constructor(
+    private val httpClient: UrlConnectionHttpClient = UrlConnectionHttpClient.getDefaultInstance()
+) {
+    companion object {
+        private const val CONFIG_BASE_URL = "https://native-ux-mock-api.azurewebsites.net/config"
```

Comment:
Please add comments describing what this URL is for

Reply:
Is this addressed?

Reply:
I have addressed it now.